lxd/api_1.0.go

```diff
@@ -1015,6 +1016,10 @@ func doAPI10UpdateTriggers(d *Daemon, nodeChanged, clusterChanged map[string]str
 		case "core.syslog_socket":
 			syslogSocketChanged = true
 		}
+
+		if strings.HasPrefix(key, "storage.project_") {
```

Comment:
just a nit, but for efficiency, we could use:

```go
switch key {
    case ...
    default:
         if strings.HasPrefix(key, "storage.project_") {
         ...
}
```

This way the `strings.HasPrefix` check wont be evaluated for the keys that exact match above.